Automattic/newspack-popups
# feat: enable excluding categories per prompt

### All Submissions:






### Changes proposed in this Pull Request:



Closes #68.

### How to test the changes in this Pull Request:

1. Create a prompt (inline or overlay), open a post from category A.
2. Observe the prompt rendering on the post.
3. Checkout this.
4. Edit the prompt, from the advanced filter on the right, exclude category A.
5. Reload the post (you need to clear cache or switch to incognito in case of overlays), and observe the prompt is no longer rendered.
6. Edit the prompt, and include category A.
7. Reload the post and observe that the prompt is not loaded (exclusion rule has priority).
8. Bonus for the two tests:
```
phpunit --filter test_categories_exclusion
phpunit --filter test_categories_exclusion_priority_over_inclusion
```

### Other information:
